main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -119,10 +119,10 @@ object CompletionProvider {
       //
       // Declarations.
       //
-      case SyntacticContext.Decl.Trait => KeywordOtherCompleter.getCompletions(context)
-      case SyntacticContext.Decl.Enum => KeywordOtherCompleter.getCompletions(context)
+      case SyntacticContext.Decl.Trait => KeywordDeclCompleter.getCompletions(context)
+      case SyntacticContext.Decl.Enum => KeywordDeclCompleter.getCompletions(context)
```

Comment:
Suppose this is inside an enum-- then what keywords are possible?

Reply:
I see. Only *one* meaningful keyword can occur here, that is `case`. At the time, I misunderstood what being in the syntactic context of enum declaration meant, which is what caused me to make this mistake :)